dockerfiles/nanocloud-backend/Dockerfile

```diff
@@ -11,6 +11,23 @@ COPY repos_nanocloud /opt/repos
 
 RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - && apt-get install -y nodejs
 
+RUN npm install -g bower
```

Comment:
I modified the commit, it should be better.

But I could have created one more comit, don't know what is the best practice on this.